File: Add Code Here/C++/gcd.cpp
#include<stdio.h>
int gcd(int a,int b);
int main()
{
	int a,b,res;
	scanf("%d %d",&a,&b);
	res=gcd(a,b);
	printf("gcd is %d",res);
}
int gcd(int a,int b)
{
	int res=0,i;
	if(a>b)
	{
		for(i=1;i<=a;i++)
		{
			if(a%i==0)
			{
				if(b%i==0)
					res=i;
			}
		}
	}
	else
	{
		for(i=1;i<=b;i++)
		{
			if(a%i==0)
			{
				if(b%i==0)
					res=i;
			}
		}
	}
	return res;
}

File: Add Code Here/JAVA/gcd.java
import java.io.*;

class GCD {
    static int gcd(int a, int b)
    {
        int i;
        if (a < b)
            i = a;
        else
            i = b;
        for (i = i; i > 1; i--) {
            if (a % i == 0 && b % i == 0)
                return i;
        }
        return 1;
    }